lib/options.py

```diff
@@ -511,8 +511,36 @@ def __init__(self):
 
         self.check()
 
+        self.correct_timeout_offset(parser)
+
         Options._initialized = True
 
+    def correct_timeout_offset(self, parser: argparse.ArgumentParser) -> None:
+        """Check and correct timeout offset if necessary."""
+        default_time_offset = 10
```

Comment:
You need to add a description of this offset (and what it is) to both the **commit message** and the **PR description**.